app/src/main/java/com/example/euriskocodechallenge/ui/home/news/NewsFragment.kt

```diff
@@ -79,7 +79,7 @@ class NewsFragment : Fragment() {
     private fun observeViewModel(view: View) {
         //Observe Retrofit Result -> Set Recycler View Data, Dismiss Progress Bar
         viewModel.response.observe(viewLifecycleOwner) {
-            if (it != null) {
+            if (!it.equals(null)) {
```

Comment:
@martin371998 avoid null checks, instead use safe calls and ?.let { }

Reply:
done in another branch